build/root/WORKSPACE

```diff
@@ -80,3 +80,11 @@ docker_pull(
     repository = "library/busybox",
     tag = "latest",  # ignored, but kept here for documentation
 )
+
+docker_pull(
+    name = "nested-kubelet-base",
+    digest = "sha256:f8527c0f16436d14bfacb192ce5522f7da73103d4e1f44f5cec2983c996e9adf",
+    registry = "gcr.io",
+    repository = "qlee-dev/nested-kubelet-base-amd64",
```

Comment:
nope nope nope.

Reply:
Per previous comments this is temporary so that the PR builds, there will be a follow up to make this use a k8s.io image. It doesn't make sense to publish this there until this is accepted.